tozny/data_source_tozny_realm_application_saml_description.go

```diff
@@ -66,8 +77,7 @@ func dataSourceRealmApplicationSAMLDescription() *schema.Resource {
 func dataSourceRealmApplicationSAMLDescriptionRead(ctx context.Context, d *schema.ResourceData, m interface{}) diag.Diagnostics {
 	var diags diag.Diagnostics
 
-	toznyClientCredentialsFilePath := d.Get("client_credentials_filepath").(string)
-	toznySDK, err := MakeToznySDK(toznyClientCredentialsFilePath, m)
+	toznySDK, err := MakeToznySDK(d, m)
```

Comment:
more elegant then my approach, just pass in all the data and don't parse it in each resource